ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
+            $( selector ).removeClass( spinclass );
+            $( selector ).addClass( spinclass );
+            $timeout(function () {
```

Comment:
Could you try the trick here https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Tips ?